src/Symfony/Component/DependencyInjection/ParameterBag/FrozenParameterBag.php

```diff
@@ -28,12 +28,14 @@ class FrozenParameterBag extends ParameterBag
      *
      * This is always the case when used internally.
      *
-     * @param array $parameters An array of parameters
+     * @param array                           $parameters An array of parameters
+     * @param ParameterResolverInterface|null $resolver
      */
-    public function __construct(array $parameters = array())
+    public function __construct(array $parameters = array(), ParameterResolverInterface $resolver = null)
     {
         $this->parameters = $parameters;
         $this->resolved = true;
+        $this->resolver = $resolver;
```

Comment:
Isn't the resolver useless here in the frozen bag ? Shoudn't everything already being resolved ?